src/App.tsx

```diff
@@ -1,17 +1,24 @@
 import { createRoot } from 'react-dom/client';
 import { reatomContext } from '@reatom/react';
 import { StrictMode } from 'react';
+import { appInit } from '~core/app/init';
 import { store } from '~core/store/store';
-import { AuthWrapper } from '~core/auth';
 import { Router } from '~core/router';
 
 const root = createRoot(document.getElementById('root')!);
-root.render(
-  <StrictMode>
-    <reatomContext.Provider value={store}>
-      <AuthWrapper>
+
+appInit().then(() => {
```

Comment:
in case of appInit error we have only blank screen
maybe it worth to add at least error text explanation ?

Reply:
we have error processing in src\index.ts